doc/whats_new/v0.23.rst

```diff
@@ -7,7 +7,7 @@
 Version 0.23.0
 ==============
 
-**In Development**
+**May 12 2020**
```

Comment:
Will this be enough time for users to test the RC (assuming it's released in the following days)?

Reply:
If these PRs get merged, I can release today hopefully, and it should give people enough time.